extension/app/ts/background/settings.ts

```diff
@@ -107,3 +108,12 @@ export function saveSimulationMode(simulationMode: boolean) {
 export function savePendingAccessRequests(pendingAccessRequests: readonly PendingAccessRequest[]) {
 	browser.storage.local.set({ pendingAccessRequests: pendingAccessRequests })
 }
+export function saveOpenedAddressBookTabId(addressbookTabId: number) {
+	browser.storage.local.set({ addressbookTabId: addressbookTabId })
+}
+
+export async function getOpenedAddressBookTabId() {
+	const tabIdData = await browser.storage.local.get(['addressbookTabId'])
```

Comment:
Seems odd to store a tab ID in local storage.  When you restart your browser, do tab IDs get reused?  Could we accidentally overwrite someone's tab by doing this?  Why not just store it in memory?

Reply:
if you store it in memory and restart the browser, the memory gets lost, so the tab gets lost too and the addressbook would not be attached to anything.

ID's in chrome and firefox seemed to be big incremental numbers, so they get reused only after long time I believe, I don't know if this is some standard or not.

Reply:
Initially I tried to just scan through all tabs and see if adressbook is open, but that requires extra access rights for the extension, and I thought I can implement it like this as well

Reply:
> if you store it in memory and restart the browser, the memory gets lost, so the tab gets lost too and the addressbook would not be attached to anything.

Tab IDs are stable across browser restarts?  What happens if you have your browser setup to *not* remember tabs across restarts?